src/shared/modules/requests/requestsDuck.ts

```diff
@@ -23,25 +23,49 @@ import bolt from 'services/bolt/bolt'
 import { APP_START } from 'shared/modules/app/appDuck'
 
 export const NAME = 'requests'
-export const REQUEST_SENT = `${NAME}/SENT`
-export const CANCEL_REQUEST = `${NAME}/CANCEL`
-export const REQUEST_CANCELED = `${NAME}/CANCELED`
-export const REQUEST_UPDATED = `${NAME}/UPDATED`
+export const REQUEST_SENT = 'requests/SENT'
+export const CANCEL_REQUEST = 'requests/CANCEL'
+export const REQUEST_CANCELED = 'requests/CANCELED'
+export const REQUEST_UPDATED = 'requests/UPDATED'
```

Comment:
Typescript struggles to correctly type the actions if they´re not simple strings